docs/deploying.md

```diff
@@ -0,0 +1,133 @@
+# Deploying
+
+Follow the guidance below to deploy changes.
+
+Each section covers a different kind of change. The aim is to help minimise risk when deploying changes to this app, but you can always choose to skip some of the steps.
```

Comment:
It might be worth putting something here saying that it’s the PDF-generating/validating/reading bits of the code that we need to be a bit more careful around. I just wouldn’t want someone new to the team thinking that they need to do this testing to deploy any change. But I will leave this up to you.

Reply:
That's fair. My assumption was a new person would see there's no applicable section for their change, but that's a bit implicit. I've rewritten this to be more like a contents list.

What do you think?